### PR TITLE
Serialise S3 message eventName correctly

### PR DESCRIPTION
**Pull request by AlexeyRaga**

### CODE REVIEW

**Comment by jianwan2016** [comment]
👍

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,11 +29,13 @@ workflows:
...
-      #     username: ${HACKAGE_USER}
-      #     password: ${HACKAGE_PASS}
+      - hackage/upload:
+          context: haskell-ci
+          publish: true
+          filters:
+            branches:
+              only: master
+          # requires:
+          #   - GitHub Release
```
Comment (newhoggy) [inline .circleci/config.yml]: Shouldn't we enable github releases too?

File: antiope-s3/src/Antiope/S3/Messages.hs [hunk 2]
```diff
@@ -63,7 +63,7 @@ instance FromJSON S3Message where
 instance ToJSON S3Message where
   toJSON msg = Aeson.object
     [ "eventTime" .= eventTime msg
-    , "eventName" .= T.intercalate ":" ["s3", eventType (eventName msg), action (eventName msg)]
+    , "eventName" .= T.intercalate ":" [eventType (eventName msg), action (eventName msg)]
```
Comment (newhoggy) [inline antiope-s3/src/Antiope/S3/Messages.hs]: Do you mean `eventType` already puts in an `s3:`?

Comment (AlexeyRaga) [reply antiope-s3/src/Antiope/S3/Messages.hs]: Looks like `s3:` shouldn't be there at all.

Comment (newhoggy) [reply antiope-s3/src/Antiope/S3/Messages.hs]: Ah ok.

File: .circleci/config.yml (hunk 1 above)
Comment (AlexeyRaga) [reply .circleci/config.yml]: I don't know how to do it now, so I kept it disabled.
I also have no time to work on GH orb right now.

**Comment by newhoggy** [comment]
👍

**Pull request merged**